shared/models/users/user.model.ts

```diff
@@ -7,6 +7,7 @@ export interface User {
   id: number
   username: string
   email: string
+  emailVerified?: boolean
```

Comment:
My thinking was since `null` is valid then optional made sense but would the better practice be `emailVerified: boolean | null`? 

Nonetheless, if I make it required I will need to update code related to `auth-user.model`. This is fine if you want me to go ahead, but it's probably not worth it if you agree with the above suggestion to make `emailVerified` an enumeration so that different states are more explicit in meaning compared to `true`/`false`/`null`. Let me know your preference and I will update this or work on next PR for changing the `emailVerified` type.

Reply:
`boolean | null` is fine :+1: